src/sentry/rules/processor.py

```diff
@@ -257,7 +257,7 @@ def apply_rule(self, rule: Rule, status: GroupRuleStatus) -> None:
                 rule_id=rule.id,
             )
 
-        notification_uuid = uuid.uuid4()
+        notification_uuid = str(uuid.uuid4())
```

Comment:
@jangjodi is this going to be 16 characters?

Reply:
@scttcper don't we have a utility function for this?

Reply:
no we switched to this